src/parameter_list_type.F90

```diff
@@ -1206,7 +1211,12 @@ subroutine get_vector_string (this, name, value, default, stat, errmsg)
     else
       if (present(default)) then
         call set_vector (this, name, default)
+#ifdef INTEL_DPD200255963
+        if (allocated(value)) deallocate(value)
+        allocate(character(len(default(1))) :: value(size(default)))
```

Comment:
should just be len(default), len is a property of the variable (including arrays)